File: app/src/main/java/com/parabola/newtone/mvp/view/ChoosePlaylistView.java
package com.parabola.newtone.mvp.view;

import com.arellomobile.mvp.MvpView;
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;
import com.parabola.domain.model.Playlist;

import java.util.List;

@StateStrategyType(AddToEndSingleStrategy.class)
public interface ChoosePlaylistView extends MvpView {
    void refreshPlaylists(List<Playlist> playlists);

    void closeScreen();
}

File: app/src/main/java/com/parabola/newtone/mvp/view/ChoosePlaylistView.kt
package com.parabola.newtone.mvp.view

import com.parabola.domain.model.Playlist
import moxy.MvpView
import moxy.viewstate.strategy.alias.AddToEndSingle

@AddToEndSingle
interface ChoosePlaylistView : MvpView {
    fun refreshPlaylists(playlists: List<Playlist>)

    fun closeScreen()
}
